tests/core/data/test_data_manager.py

```diff
@@ -70,7 +70,8 @@ def test_create_and_get_csv_data_node(self):
         assert _DataManager._get(csv_dn.id).job_ids == csv_dn.job_ids
         assert not _DataManager._get(csv_dn.id).is_ready_for_reading
         assert _DataManager._get(csv_dn.id).is_ready_for_reading == csv_dn.is_ready_for_reading
-        assert len(_DataManager._get(csv_dn.id).properties) == 3
+        assert len(_DataManager._get(csv_dn.id).properties) == 4
+        assert _DataManager._get(csv_dn.id).properties.get("default_path") is None
```

Comment:
That's the third reason we want to avoid the new behavior. We don't want to pass None attributes to entities at their creation. It is pointless.

Reply:
Resolve by the latest commit